### DISCUSSION TITLE
After install of home-brew with .pkg installer, brew not in path

### DISCUSSION THREAD

**Discussion by mrgehring** [Homebrew/discussions#4780] [Getting started] [answered]
### Output of `brew config`

```shell
command not found
```

### Output of `brew doctor`

```shell
command not found
```

### Description of issue

Moved to a new M2 Mac mini from an older Intel Mac mini. After migration, I uninstalled brew as I got an error form brew upgrade that I needed the apple silicon version. So, I uninstalled it, then installed brew from here using the 4.1.10 pkg :

https://github.com/Homebrew/brew/releases/tag/4.1.10

Afterwards, brew command in terminal did not work, presumably due to it not bing in the path. I googled and found this solution. Why does the .pkg installed not handle this?

**Comment by SMillerDev** [comment, marked as answer]
For the same reason the normal installer doesn't add this. We don't edit user settings.

The installer does tell you to do this though.
